packages/api/src/export/exportHelpers.ts

```diff
@@ -0,0 +1,132 @@
+import type { SignedInAuthObject } from "@clerk/backend/internal";
+import { clerkClient } from "@clerk/nextjs/server";
+import { LessonSnapshots } from "@oakai/core";
+import type { Snapshot } from "@oakai/core/src/models/lessonSnapshots";
+import type {
+  LessonExportType,
+  LessonSnapshot,
+  PrismaClientWithAccelerate,
+} from "@oakai/db";
+import * as Sentry from "@sentry/nextjs";
+
+import {
+  ailaGetExportBySnapshotId,
+  type OutputSchema,
+} from "../router/exports";
+
+export const getUserEmail = async (ctx: {
+  auth: SignedInAuthObject;
+  prisma: PrismaClientWithAccelerate;
+}) => {
+  const user = await clerkClient.users.getUser(ctx.auth.userId);
+  const userEmail = user?.emailAddresses[0]?.emailAddress;
+
+  return userEmail;
+};
+
+const categoryMap: Record<string, string> = {
```

Comment:
Why is exporting usefull for tests ?

Reply:
We can import the list of categories in the test file. I'm generally exporting types wherever they are defined so that when we have tests we can share the types in the test files